From c26a51a7c61fd423b031294c6de2c3cae9056c20 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 24 Jan 2019 20:52:00 +0100
Subject: [PATCH 1/1] x86: #define CONFIG_LOADADDR 0x1100000

arch/x86/dts/qemu-x86_i440fx.dts reserves memory for PCI at 0x1000000.
Loading via the `dhcp` command to this address leads to a crash on
qemu-x86_64_defconfig. So let's define CONFIG_LOADADDR as 0x1100000.

Reported-by: Alexander Graf <agraf@suse.de>
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
compatible='pci-x86' is a U-Boot specific device tree binding.
Unfortunately it is not documented. Simon, it would be helpful
if you could provide some README.
---
 include/configs/x86-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/x86-common.h b/include/configs/x86-common.h
index 4180b25f97..e936230157 100644
--- a/include/configs/x86-common.h
+++ b/include/configs/x86-common.h
@@ -99,7 +99,7 @@
 #define CONFIG_ROOTPATH		"/opt/nfsroot"
 #define CONFIG_HOSTNAME		"x86"
 #define CONFIG_BOOTFILE		"bzImage"
-#define CONFIG_LOADADDR		0x1000000
+#define CONFIG_LOADADDR		0x1100000
 #define CONFIG_RAMDISK_ADDR	0x4000000
 #if defined(CONFIG_GENERATE_ACPI_TABLE) || defined(CONFIG_EFI_STUB)
 #define CONFIG_OTHBOOTARGS	"othbootargs=\0"
-- 
2.20.1

